go.mod

```diff
@@ -17,13 +17,15 @@ require (
 	github.com/integr8ly/keycloak-client v0.0.0-20200129153447-ae34238c1f86
 	github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024
 	github.com/keycloak/keycloak-operator v0.0.0-20200207072807-b527c8b26465
+	github.com/matryer/moq v0.0.0-20200125112110-7615cbe60268 // indirect
 	github.com/openshift/api v3.9.1-0.20191031084152-11eee842dafd+incompatible
 	github.com/openshift/client-go v3.9.0+incompatible
 	github.com/openshift/cluster-samples-operator v0.0.0-20191113195805-9e879e661d71
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20191115003340-16619cd27fa5
 	github.com/operator-framework/operator-marketplace v0.0.0-20191105191618-530c85d41ce7
 	github.com/operator-framework/operator-sdk v0.15.1
 	github.com/pelletier/go-toml v1.3.0 // indirect
+	github.com/pkg/errors v0.8.1
```

Comment:
Would be nice to remove `pkg/errors` in favour of using `fmt.Errorf`  for consistency as there's been an effort by Ger to remove it's usage https://github.com/integr8ly/integreatly-operator/pull/390/files